src/lib/core/option/option.scss

```diff
@@ -0,0 +1,30 @@
+@import '../style/menu-common';
+@import '../a11y/a11y';
+
+@mixin md-option() {
```

Comment:
Add comment explaining what this mixin is for (i.e., used by both select and autocomplete)